src/stylus/fablevision/breed-button.styl

```diff
@@ -0,0 +1,27 @@
+.geniblocks.breed-button
+  background: url('../resources/fablevision/layout-c/breed_surround.png')
+  width: 370px
+  height: 369px
+  display: flex
+  justify-content: center
+  align-items: center
+
+  .clickable-button
```

Comment:
`clickable-button` implies that this is applicable to buttons in general, but since it's really breed button specific, I'd call it `breed-button` (once the current `breed-button` is renamed to `breed-button-surround`.